style.css

```diff
@@ -515,3 +523,365 @@ input[type="text"] {
+      margin-top: 30px;
+      flex-wrap: wrap;
+      margin-right: 155px;
+    }
+
+    .desk {
+      display: inline-block;
+      text-decoration: none;
+      font-family: 'Poppins', sans-serif;
+      font-size: 16px;
+      margin-right: 20px;
+      font-style: normal;
+      font-weight: 500;
+      line-height: 20px;
+      color: #344563;
+    }
+
+    .menu {
+      display: none;
+    }
+
+    .nav-container {
+      display: flex;
+      flex-wrap: wrap;
+    }
+
+    .ruby {
+      display: inline-block;
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      background: #ebebff;
+      border-radius: 8px;
+      color: #6070ff;
+      font-weight: 222px;
+      font-size: 12px;
+      background-color: #ebebff;
+      font-family: 'Poppins', sans-serif;
+      padding: 8px 12px;
+      margin-right: 10px;
+    }
+
+    .part1 {
+      background: url(images/5.png);
+      background-position: cover;
+      background-repeat: no-repeat;
+      background-size: 100% 100%;
+      margin-top: 50px;
+    }
+
+    .head-name {
+      font-family: 'Poppins', sans-serif;
+      font-style: normal;
+      font-weight: 700;
+      font-size: 48px;
+      line-height: 60px;
+      display: flex;
+      align-items: center;
+      letter-spacing: 0.37px;
+      color: #091e42;
+      margin-left: 260px;
+      margin-top: 300px;
+    }
+
+    .head-para {
+      display: flex;
+      width: 920px;
+      font-family: 'Poppins', sans-serif;
+      font-style: normal;
+      font-weight: 400;
+      font-size: 20px;
+      line-height: 28px;
+      color: #344563;
+      margin-left: 260px;
+    }
+
+    .sub-head {
+      margin-left: 260px;
+    }
+
+    .social-media {
+      margin-left: 230px;
+    }
+
+    .logo {
+      margin-left: 155px;
+    }
+
+    .card {
+      width: 80%;
+      height: 400px;
+      border-radius: 24px;
+      display: flex;
+      flex-direction: row;
+      justify-content: center;
+      background: #fff;
+      border: 1px solid #dfe1e6;
+      padding: 10px;
+      padding-left: 2%;
+      margin-left: 8%;
+    }
+
+    .pic {
+      order: 2;
+    }
+
+    .card img {
+      width: 95%;
+      height: 95%;
+    }
+
+    .title {
+      width: 367px;
+      height: 46px;
+      font-family: 'Poppins', sans-serif;
+      font-style: normal;
+      font-weight: 700;
+      font-size: 40px;
+      line-height: 52px;
+      margin-left: 46px;
+      margin-top: 27px;
+      margin-right: 46px;
+      margin-bottom: 21px;
+    }
+
+    .field {
+      width: 297px;
+      height: 33px;
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      padding: 0;
+      gap: 12px;
+      margin-left: 42px;
+    }
+
+    .trainnin {
+      padding: -43px;
+      margin-top: 19px;
+      margin-left: -17px;
+    }
+
+    .about-myself {
+      display: grid;
+      grid-template-columns: 1fr 1fr;
+      height: 668px;
+      margin-left: 10px;
+    }
+
+    .form {
```

Comment:
In your contact section

- Kindly make your background image responsive to fit the whole width of your screen in a different size. 
- Kindly center your form also in your different size after 768px of width

![image](https://user-images.githubusercontent.com/93630700/229524219-6e886dbc-07a8-4470-8b8f-a955fa3f0164.png)